interface/VideoCommonDefs.h

```diff
@@ -69,6 +69,8 @@ extern "C" {
          ((uint32_t)(uint8_t)(ch2) << 16)  | ((uint32_t)(uint8_t)(ch3) << 24))
 
 #define YAMI_FOURCC_NV12 YAMI_FOURCC('N','V','1','2')
+#define YAMI_FOURCC_P010 YAMI_FOURCC('P','0','1','0')
+#define YAMI_FOURCC_I010 YAMI_FOURCC('I','0','1','0')
```

Comment:
it is OK to only use P010. For I010, it is OK to use by YAMI_FOURCC('I','0','1','0')